### ISSUE TITLE
Mail Settings - Password is Missing Upon Page Reload

### ISSUE THREAD

**Issue by shefik** [zikula/core#3315]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.4
| PHP Version       | 5.6

### Expected behavior

The password should be populated in the form field `/mailer/config/config` under the login for `SMTP settings`.

### Actual behavior

The password is missing, even though it is correctly stored in the YAML file.

### Steps to reproduce

Go to `/mailer/config/config` and save SMTP settings.

**Comment by Guite** [comment]
This is on purpose: http://symfony.com/doc/current/reference/forms/types/password.html#always-empty

**Comment by Guite** [comment]
Just saw we set this to `false` in the config form type.

**Comment by shefik** [comment]
But, if it's on purpose to remove the password, it's still being exposed at `Current settings` at `/mailer/config/test`.

**Comment by Guite** [comment]
Closing as `won't fix` because this is how the `PasswordType` works.
We set `always_empty` to `false` but this only affects displaying the form after submit. So in case you entered some invalid data you won't have to enter the password again.
But the `always_empty` option does not have any effect when displaying the form before submit.

See https://github.com/symfony/symfony/blob/5129c4cf7e294b1a5ea30d6fec6e89b75396dcd2/src/Symfony/Component/Form/Extension/Core/Type/PasswordType.php#L26
